### DISCUSSION TITLE
Imports for Storybook

### DISCUSSION THREAD

**Discussion by vsirenko** [storybookjs/storybook#24883] [Help] [answered]
Good morning, guys.

I have an error when I try to import my icon, if I import it directly everything is fine.

<img width="412" alt="image" src="https://github.com/storybookjs/storybook/assets/81309430/e655bb9c-cb0a-43b3-b541-eb6b902ecda7">

But if I try to import from the list I get an error: Loading chunk [chunkName] failed.
<img width="486" alt="image" src="https://github.com/storybookjs/storybook/assets/81309430/de760764-b2f9-40dc-ab6b-8337b040acc2">

`components/icons`

<img width="277" alt="image" src="https://github.com/storybookjs/storybook/assets/81309430/c25faa1a-add5-4b0e-bf5a-3a37ed8b0c83">

`components/icons/mono`
<img width="643" alt="image" src="https://github.com/storybookjs/storybook/assets/81309430/f82d8fe8-c55e-4dc6-9b18-9f30adc3022d">

`Icon component`

<img width="750" alt="image" src="https://github.com/storybookjs/storybook/assets/81309430/2b43424e-7123-42ab-88e1-d1c9f0cbd7df">

I think the problem is related to aliases, and my local congfig, I would be grateful if someone could tell me how to customize the imports

(aliases work correctly for everything else).

**Comment by vsirenko** [comment, marked as answer]
Solved with the addition of the config typescript 
  `allowSyntheticDefaultImports: true`

**Reply by joevaugh4n** [reply]
Nice work, @vsirenko!
